banzai/tic-tac-toe.rb

```diff
@@ -0,0 +1,108 @@
+# this gem injects a color method to the string when it is displayed by console
+require 'colorize' 
+
+@level = 2
+
+def draw_field(val, index)
+  (val.is_a? Integer) ? (index + 1 < 10 ? " #{val}" : val) : " #{val}"
+end
+
+def draw_cell(cell, index)
+  return draw_field(index + 1, index).to_s.white if cell == 0
+  return draw_field("O", index).yellow if cell == -1
+  return draw_field("X", index).blue if cell == 1
+end
+
+def draw_tic(arr)
+  size_line = 5 * @level + 1
+  line, col = "-".green, "|".green
+  border = line * size_line
+  total = border
+  row = col
+  arr.each_with_index do |cell, index|
+    content = draw_cell(cell, index)
+    row = row + " #{content} #{col}"
+    if ((index + 1) % @level == 0)
+      total = total + "\n" + row + "\n" + border
+      row = col
+    end
+  end
+  puts total
+end
+
+def finish_game?(arr)
+  arr.include?(0)
```

Comment:
I like how you created a super simple function to give meaning to a normal operation, better for understanding the code later 👍 
One think you could do is add a simple comment to explain the **why** of this, why it should finish if `arr.include?(0)`